lib/backend/model/bgppeer.go

```diff
@@ -32,6 +32,31 @@ var (
 	typeBGPPeer        = reflect.TypeOf(BGPPeer{})
 )
 
+type SelectorBGPPeerKey struct {
+	Name string `json:"-" validate:"omitempty"`
+}
+
+func (key SelectorBGPPeerKey) defaultPath() (string, error) {
+	e := fmt.Sprintf("/calico/bgp/v1/global/peer_sel/%s", key.Name)
```

Comment:
Is this intentionally still in the v1 space?  (That feels surprising to me.)

Reply:
Yeah, it's because confd only speaks the "v1" api over the syncer, so we need to convert the v3 stuff into something it understands (even though this obviously wasn't a part of the original v1 API and will never show up in etcd with a /v1 key)